src/mc-range-encoding.c

```diff
@@ -190,7 +191,7 @@ mc_getTypeInfoDouble (mc_getTypeInfoDouble_args_t args,
       return false;
    }
 
-   if (isinf (args.value) || isnan (args.value)) {
+   if (mc_isinf (args.value) || mc_isnan (args.value)) {
       CLIENT_ERR ("Infinity and Nan double values are not supported.");
```

Comment:
```suggestion
      CLIENT_ERR ("Infinity and NaN double values are not supported.");
```

I know this wasn't part of your change, but I think the error message requires this tweak.